backend/route/chat/general.js

```diff
@@ -1,11 +1,26 @@
-const io = require("socket.io");
+let io = require("socket.io");
 
-module.exports = function(app,mongoose,server){
-    const socket = io(server);
+module.exports = function (app, mongoose, server) {
+    io = io(server);
```

Comment:
is it better to reassign io, and not have separate names for socket, io?
I'm ok with either one, but was wonder if there was a conscious reasoning behind this decision

Reply:
It is better to have seperate names, but I realized from many examples that people usually name the other variable ( I forgot how it is called. io.on("connection", *THIS VARIABLE*=>{}) socket, which is also contextually more true. To keep this pattern, I reassigned io.